modules/smart-agent_health-checker/conf/01-health-checker-status.yaml

```diff
@@ -0,0 +1,14 @@
+module: health-checker
+name: status
+id: health_checker_status
+
+transformation: true
+aggregation: false
```

Comment:
It is better to provide a variable to let the user customize this depending on his environment.

Reply:
done